openedx/xblock-drag-and-drop-v2
# style: improves feedback and popup

### Description

- Removes dots on feedback and moves final feedback to end and set bold text on assessment mode
- Improves css styles for popup feedback
- Removes special popup feedback styles for mobile
- Improves feedback message when you reach the limit of elements on a zone
- Fixes the issue dragging an image item on Safari mobile
- Fixes width issue for zone container on Safari mobile

### Testing instructions
To test the popup improvements:
1. Use the standard demo of drag and drop v2
2. Drop the incorrect element into a zone
![Captura de Pantalla 2022-03-06 a la(s) 1 02 04 p  m](https://user-images.githubusercontent.com/363698/156931211-c4d232c4-5940-43fc-8279-3b69822dadc2.png)
4. Drop the correct element into a zone
 ![Captura de Pantalla 2022-03-06 a la(s) 1 02 22 p  m](https://user-images.githubusercontent.com/363698/156931230-ba01ea71-0a85-4575-8ab3-4f411f86aff8.png)
6. Repeat the steps using the mobile view tool on browser

To test the message when you reach limit:
1. Use the standard demo of drag and drop v2
2. Change the mode from standard to assessment
3. Set the limit on elements on a zone to 1
4. Drop an item into a zone
7. Drop a second element into the same zone
![Captura de Pantalla 2022-03-06 a la(s) 1 12 32 p  m](https://user-images.githubusercontent.com/363698/156931539-8bada1d6-57b3-4b35-a170-812d6820402c.png)
8. Repeat the steps on mobile view

To test final feedback on assessment mode and removed dots:
1. Use the standard demo of drag and drop v2
2. Change the mode from standard to assessment
3. Set the maximum of attempts to 2
4. Drop an item into the correct zone and 2 into incorrect zones
5. Press submit and see the popup
![Captura de Pantalla 2022-03-06 a la(s) 1 15 57 p  m](https://user-images.githubusercontent.com/363698/156931644-07313dc7-6517-4067-b7bd-e8943ca0fc53.png)
6. Press submit again and see the feedback under the drag and drop area
![Captura de Pantalla 2022-03-17 a la(s) 6 12 04 p  m](https://user-images.githubusercontent.com/363698/158896068-cc3f7bf0-4638-44f4-9bc9-d159462b5f87.png)
7. Repeat the steps for mobile
